src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -51,39 +52,78 @@ public class TurretSubsystem extends SubsystemBase {
   private CANSparkMax turretMotor;
   private SingleJointedArmSim turretSim;
   private Mechanism2d turretMech;
-  private MechanismRoot2d root;
+  private MechanismRoot2d turretRoot;
   private MechanismLigament2d turretAngleSim;
   private ShuffleboardTab turretTab = Shuffleboard.getTab("Turret Tab");
-  private SimDeviceSim simEncoder;
+  private SimDeviceSim simTurretEncoder;
   private SimDouble simTurretPos;
 
   private PIDController turretPIDController;
   private CANcoder highGearCANcoder;
   private CANcoder lowGearCANcoder;
   private double turretPosition;
   private double turretVelocity;
-  private Consumer<Double> testPosition;
+  private Consumer<Double> turretTestPosition;
   private ShuffleboardTab configTab = Shuffleboard.getTab("Config");
-  private GenericEntry kP = configTab.add("Turret KP", Constants.TurretConstants.KP).getEntry();
-  private GenericEntry kI = configTab.add("Turret KI", Constants.TurretConstants.KI).getEntry();
-  private GenericEntry kD = configTab.add("Turret KD", Constants.TurretConstants.KD).getEntry();
+  private GenericEntry turretKP =
+      configTab.add("Turret KP", Constants.TurretConstants.TURRET_KP).getEntry();
+  private GenericEntry turretKI =
+      configTab.add("Turret KI", Constants.TurretConstants.TURRET_KI).getEntry();
+  private GenericEntry turretKD =
+      configTab.add("Turret KD", Constants.TurretConstants.TURRET_KD).getEntry();
   private DebugEntry<Double> turretPositionEntry =
-      new DebugEntry<Double>(turretPosition, "Position", this);
+      new DebugEntry<Double>(turretPosition, "Turret Position", this);
   private DebugEntry<Double> turretGoalPositionEntry =
-      new DebugEntry<Double>(0.0, "Goal Position", this);
+      new DebugEntry<Double>(0.0, "Turret Goal Position", this);
   private DebugEntry<Double> turretVelocityEntry =
-      new DebugEntry<Double>(turretVelocity, "Velocity", this);
+      new DebugEntry<Double>(turretVelocity, "Turret Velocity", this);
+
+  private CANSparkMax pitchMotor;
```

Comment:
This could be made `final`, and I presume that most of the rest of these attributes could be, as well.